Gemfile

```
@@ -12,4 +12,5 @@ gem 'asciidoctor-pdf', '1.5.3'
 gem 'coderay', '1.1.3'
 gem 'pygments.rb', '1.2.1'
 gem 'thread_safe', '0.3.6'
-gem 'epubcheck', '3.0.1'
+gem 'epubcheck-ruby'
+gem 'html-proofer'
```

```suggestion
gem 'epubcheck-ruby', '4.2.4.0'
gem 'html-proofer', '3.17.2'
```

I think it's a good idea to pin these dependencies as well, just like we're doing for all the other dependencies.

Can you check if this is right? I've grabbed the latest versions from the rubygems.org site.

---

It is right. But I'm not sure that we want to rely on a pinned version of our checkers. Another point is the number of errors coming from these checks...